gpflow/quadrature.py

```diff
@@ -80,15 +80,18 @@ def mvnquad(func, means, covs, H: int, Din: int=None, Dout=None):
     return tf.reduce_sum(fX * wr, 0)
 
 
-def ndiagquad(funcs, H: int, Fmu, Fvar, **Ys):
+def ndiagquad(funcs, H: int, Fmu, Fvar, logspace: bool=False, **Ys):
     """
     Computes N Gaussian expectation integrals of one or more functions
     using Gauss-Hermite quadrature. The Gaussians must be independent.
 
-    :param funcs: Callable or Iterable of Callables that operates elementwise
+    :param funcs: the integrand(s):
```

Comment:
Type information should be in annotations, in docstring descriptions what it does. This `...: ...:` format is a bit odd.

Reply:
I know - how would I write that it can be a function, or an iterable of functions, each of which takes some number of elements (not defined, but same as length of Fmu / Fvar)? Similar problem to your other comment...